src/test/java/com/example/moviemanager/controller/MovieControllerTest.java

```diff
@@ -0,0 +1,159 @@
+package com.example.moviemanager.controller;
+
+import com.example.moviemanager.repository.Movie;
+import com.example.moviemanager.service.MovieNotFoundException;
+import com.example.moviemanager.service.MovieService;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.hamcrest.Matchers.blankString;
+import static org.hamcrest.Matchers.containsString;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@WebMvcTest
+public class MovieControllerTest {
+
+    @MockBean
+    private MovieService movieService;
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @Test
+    void returns_list_of_all_movies() throws Exception {
+        // given
+        List<Movie> movies = new ArrayList<>();
```

Comment:
I would simply initialize whole list with one statement:
`List<Movie> movies  = List.of(

    new Movie(...),
    new Movie(...)
);`

Reply:
Fixed here and in `MovieServiceTest` class